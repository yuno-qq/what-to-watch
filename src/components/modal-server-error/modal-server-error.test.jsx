import React from "react"
import {describe, test, expect} from "@jest/globals"
import renderer from "react-test-renderer"

import ModalServerError from "./modal-server-error.jsx"


describe(`<ModalServerError> snapshots:`, () => {
  test(`should render correctly`, () => {
    const tree = renderer.create(<ModalServerError />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
