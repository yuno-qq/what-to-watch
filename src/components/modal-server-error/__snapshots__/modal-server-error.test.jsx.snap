// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModalServerError> snapshots: should render correctly 1`] = `
<div
  className="modal"
>
  <div
    className="modal__content"
  >
    <h2>
      Произошла ошибка
    </h2>
    <p>
      Пожалуйста, перезагрузите страницу
    </p>
  </div>
</div>
`;
